web/packages/teleport/src/Main/Main.tsx

```diff
@@ -265,12 +266,31 @@ function FeatureRoutes({ lockedFeatures }: { lockedFeatures: LockedFeatures }) {
   return <Switch>{routes}</Switch>;
 }
 
-export const ContentMinWidth = styled.div`
-  min-width: 1250px;
-  display: flex;
-  flex-direction: column;
-  flex: 1;
-`;
+// This context allows children components to disable this min-width in case they want to be able to shrink smaller.
+type MinWidthContextState = {
+  setEnforceMinWidth: (enforceMinWidth: boolean) => void;
+};
+
+export const ContentMinWidthContext = createContext<MinWidthContextState>(null);
```

Comment:
Could we export `useContentMinWithContext` instead of `ContentMinWidthContext`?

It would be more convenient to use.